numpy/core/src/multiarray/common.h

```diff
@@ -205,7 +205,11 @@ npy_is_aligned(const void * p, const npy_uintp alignment)
      * This test is faster than a direct modulo.
      * Note alignment value of 0 is allowed and returns False.
      */
+#ifdef NPY_HAVE_NEON
```

Comment:
Sorry, perhaps I was unclear - i was suggesting you fold this into a macro within `einsum.c.src`, not edit the definition of `is_aligned`